webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBarActions.js

```diff
@@ -35,7 +35,11 @@ export const loadSwitcherResourcesByResource = (
   resource,
   { page = 1, searchQuery = '', perPage = 10 } = {}
 ) => async dispatch => {
-  const { resourceUrl, nameField, switcherItemUrl } = resource;
+  const { resourceUrl, nameField } = resource;
+  let { switcherItemUrl } = resource;
+  if (switcherItemUrl[switcherItemUrl.length - 1] === '/') {
```

Comment:
isnt this better given we are in ES6 context?
```suggestion
  if (switcherItemUrl.endsWith('/')) {
```

Reply:
Yup, it looks better, thanks